screens/AboutScreen.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/native';
+import React, { useState } from "react";
+import { Text, StyleSheet, View } from "react-native";
+
+
+const AboutContainer = styled.View`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  margin-top: 15px;
+  height: 100%;
+`;
+
+export const About = () => {
+  const titleText = "We are Techers";
```

Comment:
I really liked the fact that you declared and passing in props in the AboutContainer!